src/main/java/de/photon/anticheataddition/util/minecraft/world/ModernWorldUtil.java

```diff
@@ -166,6 +168,15 @@ public boolean areChunksLoadedBetweenLocations(Location one, Location two)
         return true;
     }
 
+    private boolean isLastChunkEqual(int lastChunkX, int lastChunkZ, int chunkX, int chunkZ) {
+        return lastChunkX == chunkX && lastChunkZ == chunkZ;
+    }
+
+    private boolean isLastLastChunkEqual(int lastLastChunkX, int lastLastChunkZ, int chunkX, int chunkZ) {
```

Comment:
This method is totally obsolete. It is the exact same method as isLastChunkEqual. Even if you removed that method and just used the one above, this still requires you to add in the four values as parameters and I think it can therefore be inlined.